modules/pipeline/providers/edgepipeline_register/interface.go

```diff
@@ -40,9 +40,18 @@ type Interface interface {
 
 	ShouldDispatchToEdge(source, clusterName string) bool
 	GetEdgeBundleByClusterName(clusterName string) (*bundle.Bundle, error)
+	ClusterIsEdge(clusterName string) bool
 	//RegisterEdgeToDialer(ctx context.Context)
 }
 
+func (p *provider) ClusterIsEdge(clusterName string) bool {
```

Comment:
unclear function.
what's the difference between `IsEdge` and this?
can you use `ShouldDispatchToEdge`?

Reply:
IsEdge determines whether the current cluster is an edge cluster. The data is obtained from the configuration file.

ClusterIsEdge determines whether a cluster is an edge cluster, and the data is queried.

ShouldDispatchToEdge determines whether the source and cluster need to be synchronized to the edge. Then, if the cluster is the current cluster, it returns false